input/lightcurves/mrk421_1996ApJ...460..644S.ecsv

```diff
@@ -0,0 +1,77 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: time, unit: MJD, datatype: float64}
+# - {name: time_min, unit: MJD, datatype: float64}
+# - {name: time_max, unit: MJD, datatype: float64}
+# - {name: flux, unit: Crab, datatype: float32}
+# - {name: flux_err, unit: Crab, datatype: float32}
+# - {name: telescope, unit: none, datatype: string}
+# - {name: paper, unit: none, datatype: string}
+# meta: !!omap
+# - COMMENT: [Mrk 421]
+time time_min time_max flux flux_err telescope paper
+48705.1757 48705.134 48705.2174 0.57 0.29 Whipple 1996ApJ...460..644S
```

Comment:
The telescope and paper is always the same.
I would suggest to put such info that's the same for the whole table in the table metadata.

Here you have an example how I put the info for SEDs:
https://github.com/gammapy/gamma-cat/blob/master/input/papers/2005/2005A%2526A...435L..17A/tev-000079.ecsv#L9
I think having the `paper_id` and `source_id` in `meta` would be great.
This will allow us to put the info with the right source and link to the paper from Python scripts easily.